routes/api.php

```diff
@@ -21,6 +21,11 @@
         });
         Route::prefix('/facility')->group(function () {
             Route::get('/list', [SystemController::class, 'facilityList']);
+            Route::prefix('/{facility}')->group(function () {
```

Comment:
endpoint /api/facility/{id}/meeting/list, nie /api/system/facility/{id}/meeting/list

Reply:
/v1/ w domyśle, w sumie nie wiem jak bardzo to zaciemnia, że podaję bez tego

Reply:
Ach dobra, zaplątałem się w tych nestowaniach. Zrobione.